src/scroll_list/PlaceholderRenderer.js

```diff
@@ -459,6 +470,28 @@ define(function(require) {
             while (element.firstChild) {
                 element.removeChild(element.firstChild);
             }
+        },
+
+        getPlaceholderContainer: function() {
+            if (this._placeholderContainer) {
+                return this._placeholderContainer;
+            }
+
+
+            var scrollList = this._scrollList;
+            var listMap = scrollList.getListMap();
+            var transformationPlane = listMap.getTransformationPlane();
+            var placeholderContainer = transformationPlane.querySelector('.placeholder-container');
+            if (placeholderContainer) {
+                return placeholderContainer;
+            }
+
+            placeholderContainer = document.createElement('div');
+            placeholderContainer.className = 'placeholder-container';
```

Comment:
Renamed to `scrollList-placeholderContainer`. The AwesomeMap is unaware of what we're doing here :)